main.py

```diff
@@ -0,0 +1,59 @@
+from house_package_saachi.data_ingestion import (
+    fetch_housing_data,
+    load_housing_data
+)
+from house_package_saachi.data_preprocessing import preprocess_housing_data
+from house_package_saachi.model_training import (
+    train_linear_regression,
+    train_decision_tree,
+    train_random_forest,
+    evaluate_model,
+)
+from house_package_saachi.model_scoring import score_model
+from house_package_saachi.utils import stratified_split
+import pandas as pd
```

Comment:
I see you have removed `utilts.py`. So modify this _import statement_ as well accordingly.

Reply:
- Made changes in main.py and run the file successfully.
- Made changes in pyproject.toml which was required to run the file.